tools/fast/action_client.go

```diff
@@ -44,7 +44,16 @@ func (f *Filecoin) ClientProposeStorageDeal(ctx context.Context, data cid.Cid,
 	sAsk := fmt.Sprintf("%d", ask)
 	sDuration := fmt.Sprintf("%d", duration)
 
-	if err := f.RunCmdJSONWithStdin(ctx, nil, &out, "go-filecoin", "client", "propose-storage-deal", sMiner, sData, sAsk, sDuration); err != nil {
+	// RunCmd does not allow empty arguments (e.g. in the cast that allowDuplicates is false, use an
```

Comment:
I think you can just do `--allow-duplicates=true` and  `--allow-duplicates=false`

Reply:
also cast -> case if we keep this comment

Reply:
We should probably move this to the other pattern by using a variadic argument for flags.

@shannonwells good example: https://github.com/filecoin-project/go-filecoin/blob/master/tools/fast/action_miner.go#L18-L36

You can then add a function in https://github.com/filecoin-project/go-filecoin/blob/master/tools/fast/process_action_options.go